src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -26,126 +27,392 @@ function oneYearFromNow() {
   return Date.now() + oneYearInMS;
 }
 
+function wait(timeout) {
+  return new Promise(resolve => {
+    setTimeout(resolve, timeout);
+  });
+}
+
 describe('ApplyForBenefits component', () => {
   let view;
-  it('renders the correct content when there are no applications in progress', () => {
-    const initialState = {
-      user: {
-        profile: {
-          savedForms: [],
+  describe('Applications in progress', () => {
```

Comment:
The first part of this diff is just nesting the existing tests in a new `describe` block...